puppetboard/templates/_macros.html

```diff
@@ -56,6 +56,9 @@
     "search": {"regex": true},
     // Default sort
     "order": [[ 0, "desc" ]],
+    //Sort ipadresses
```

Comment:
NIT: Typo in address and add space to match with line 57.   `// Sort ip address`